boards/shields/st7789v_generic/doc/index.rst

```diff
@@ -0,0 +1,57 @@
+.. _st7789v_generic:
+
+Generic ST7789V Display Shield
+##############################
+
+Overview
+********
+
+This is a generic shield for display shields based on ST7789V display
+controller. More information about the controller can be found in
+`ST7789V Datasheet`_.
+
+Pins Assignment of the Generic ST7789V Display Shield
+=====================================================
+
++-----------------------+---------------------------------------------+
+| Arduino Connector Pin | Function                                    |
++=======================+=============================================+
+| D8                    | ST7789V Reset                               |
++-----------------------+---------------------------------------------+
+| D9                    | ST7789V DC       (Data/Command)             |
```

Comment:
While this is OK, if you really want to align the description part (in parentheses), you should add another column under the heading bar:
```
+-----------------------+---------------------------------------------+
| Arduino Connector Pin | Function                                    |
+=======================+================+============================+
| D8                    | ST7789V Reset  |                            |
+-----------------------+----------------+----------------------------+
| D9                    | ST7789V DC     | (Data/Command)             |
```
etc.